src/polish.c

```diff
@@ -134,10 +134,13 @@ static c_int iterative_refinement(OSQPWorkspace *work,
                                   LinSysSolver  *p,
                                   c_float       *z,
                                   c_float       *b) {
+  c_int i, j, n, exitflag;
+  c_float *dz;
+  c_float *rhs;
+
+  exitflag = 0;
+
   if (work->settings->polish_refine_iter > 0) {
-    c_int i, j, n, exitflag = 0;
-    c_float *dz;
-    c_float *rhs;
```

Comment:
By mixing variables declarations and assignments I meant what I just changed in this commit. Here `exitflag=0` was assigned before declaring `dz` which should not be allowed in ansi C.